src/main/java/org/folio/search/service/consortium/ConsortiumSearchHelper.java

```diff
@@ -51,7 +51,7 @@ public QueryBuilder filterQueryForActiveAffiliation(QueryBuilder query, Resource
   public QueryBuilder filterQueryForActiveAffiliation(QueryBuilder query, ResourceType resource,
                                                       String contextTenantId) {
     var centralTenantId = consortiumTenantService.getCentralTenant(contextTenantId);
-    if (centralTenantId.isEmpty()) {
+    if (centralTenantId.isEmpty() || resource.isLinkedDataResource() && centralTenantId.get().equals(contextTenantId)) {
       return query;
     }
```

Comment:
The PR description mentions that you should ignore the `shared` filter in consortia mode. But here, I can see that it's ignored only for central tenant requests. Where is inaccuracy?

Reply:
The Linked Data module is proposed to be installed/activated in a Central tenant only, thus there is no difference for it. And this check is jsut additional safety check. But let me clarify PR description, thank you